lib/almanac/weather_data.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Almanac.WeatherData do
+  @current_obs_url Application.get_env(:almanac, :current_obs_url)
```

Comment:
Put import statements at the top of the module, and let the @constant declarations follow after. Separate the two by one blank line